docs/resources/azure_virtual_network_peering.md

```diff
@@ -0,0 +1,91 @@
+---
+title: About the azure_virtual_network_peering Resource
+platform: azure
+---
+
+# azure_virtual_network_peering
+
+Use the `azure_virtual_network_peering` InSpec audit resource to test properties related to a peering for a given virtual network.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`resource_group`, `vnet` and `name` or the `resource_id` must be given as a parameter.
```

Comment:
```suggestion
`resource_group`, `vnet` and either `name` or `resource_id` are required parameters.
```
Is this correct--this resource has a total of 3 required parameters?

Reply:
fixed